digest/glossary.js

```diff
@@ -0,0 +1,28 @@
+const utils = require('./utils')
+
+module.exports = () =>
+  utils.readdir('/glossary')
+    .then(convertGlossaryFileNamesToTerms)
+    .then(indexById)
+
+const convertGlossaryFileNamesToTerms = glossaryFileName =>
+  glossaryFileName
+    .map(fileName => ({
+      fileName,
+      id: fileName,
+      value: fileName.replace(/-/g, ' '),
```

Comment:
let's extract it from the top level heading of the markdown file. This means you'll need to create a `Promise.all` if terms mapped to a function that reads the `README.md` file, uses the markdown lexer to parse it, extracts the level one head text value and puts it on the term under the `value` property.

Reply:
So, to clarify, we are establishing value's property by calling one helper (readMarkdownFile) to read the markdown file, and then call another helper (extractListFromSection) to extract the title from the markdown file?